libsrc/grabber/x11/X11Grabber.cpp

```diff
@@ -72,8 +72,13 @@ void X11Grabber::setupResources()
 		_shminfo.readOnly = False;
 		XShmAttach(_x11Display, &_shminfo);
 	}
+
```

Comment:
Fixes a bug in X11 grabber. Image resampler needs to be reinitialized during resource setup